src/main/java/persistence/sql/context/impl/DefaultPersistenceContext.java

```diff
@@ -12,22 +12,16 @@
 public class DefaultPersistenceContext implements PersistenceContext {
     private final Map<KeyHolder, EntityEntry> context = new HashMap<>();
 
-    private final EntityPersister persister;
-
-    public DefaultPersistenceContext(EntityPersister persister) {
-        this.persister = persister;
-    }
-
     @Override
     public <T> EntityEntry addEntry(T entity, Status status, EntityPersister entityPersister) {
```

Comment:
```suggestion
    public <T> EntityEntry addEntry(T entity, Status status) {
```
여기있는 `EntityPersister` 도 현재는 필요 없어 보이네요 :)